src/main/scala/xiangshan/frontend/IFU.scala

```diff
@@ -814,11 +802,7 @@ class NewIFU(implicit p: Parameters) extends XSModule
   io.toIbuffer.bits.pc          := f3_pc
   io.toIbuffer.bits.ftqOffset.zipWithIndex.map{case(a, i) => a.bits := i.U; a.valid := checkerOutStage1.fixedTaken(i) && !f3_req_is_mmio}
   io.toIbuffer.bits.foldpc      := f3_foldpc
-  io.toIbuffer.bits.exceptionType := (0 until PredictWidth).map(i => MuxCase(ExceptionType.none, Seq(
-    (f3_pf_vec(i) || f3_crossPageFault(i)) -> ExceptionType.ipf,
-    (f3_gpf_vec(i) || f3_crossGuestPageFault(i)) -> ExceptionType.igpf,
-    f3_af_vec(i) -> ExceptionType.acf
-  )))
+  io.toIbuffer.bits.exceptionType := f3_exception_vec
```

Comment:
This introduced a bug: cross-page RVI instruction (the former page has no exception, while the latter has pf/gpf) is not marked as pf/gpf, #3386 fixes this.